src/jobs/backup.yml

```diff
@@ -0,0 +1,14 @@
+description: |
+    Generate a backup of production on deployment to staging.
```

Comment:
Is there a way to have it backup any install we want? I could see that being useful for feature sites maybe?

Reply:
Right now overriding the WPE_PRODUCTION_INSTALL variable would allow for backing up any install.